.github/scripts/tests/tests.sh

```diff
@@ -13,6 +13,8 @@ if [ "$GIT_WORKSPACE" = "" ]; then
 fi
 
 CLEAN_INFRA=false
+SKIP_DEPLOY=false
+SKIP_CLEANUP=false
```

Comment:
When can this actually be set to `true`? Maybe something like this?
```suggestion
SKIP_DEPLOY=${SKIP_DEPLOY:-false}
SKIP_CLEANUP=${SKIP_CLEANUP:-false}
```